Apps/JobScheduler/src/Startup.cs

```diff
@@ -130,7 +149,58 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 },
             });
 
-            app.UseMvc();
+            // This is needed if running behind a reverse proxy
+            // like ngrok which is great for testing while developing
+            app.UseForwardedHeaders(new ForwardedHeadersOptions
+            {
+                RequireHeaderSymmetry = false,
```

Comment:
We have this in the Common startup... might need to add the header symmetry

Reply:
i will look at what we did in common.